report/5.Solution/FromScreenToMotors.tex

```diff
@@ -1,85 +1,77 @@
 \section{Mapping between computer vision and movement}
 Two functions are needed to translate the screen position into a velocity on the motors.
-First we need to transfer the screen position to the USB transfer, and then that format to a motor velocity.\todo{Should this be velocity or speed?}
+The first task is to translate the screen position into a format usable for the USB transfer, and then transform that format into a motor velocity.\todo{Should this be velocity or speed?}
 
 \subsection{Screen to motors}
-In order to translate the position on the screen to a velocity two functions are needed.
-First we need to transfer the screen position to the USB transfer, and then that format it to a motor velocity.
+To translate the position on the screen into a velocity two functions are needed.
+First of all, it is necessary to make a function maps the numbers correctly. \todo{Which numbers precisely?}
 
-We need to make a function on the python part that maps these numbers.
+This function will require a 2d vector named \textit{pos} as input. \todo{what is pos}
 
-This function will take in 'pos` which is going to be a 2d vector.
+The next goal is to take a location on the screen and translate it into a 8-bit value. \todo{why} 
```

Comment:
We translate it to an 8 bit value for the T target location no?

Reply:
```
        self.endpoint.write(bytes([int(data.location.x) & 0xFF,
                                   int(data.location.y) & 0xFF,
                                   int(data.status.value[0]) & 0xFF,
                                   0]))
```

Teknisk set 2 8bit værdier.

Men jeg forstår ikke lisbys kommentar